scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# fail if some commands fails
+set -e
+# show commands
+set -x
+
+make bin
+export PATH="$PATH:$(pwd)"
+export ARTIFACTS_DIR="/tmp/artifacts"
+export CUSTOM_HOMEDIR=$ARTIFACTS_DIR
+
+kam version
+
+if [ "${ARCH}" == "s390x" ]; then
+    # Integration tests
+
+    # E2e tests
+elif  [ "${ARCH}" == "ppc64le" ]; then
+    # Integration tests
+
+    # E2e tests
+else
+    # Integration tests
+
+    # E2e tests
+fi
```

Comment:
Please remove from line 15 to line 27. These are the platform supported condition and till now ```kam``` does not release binary for such distribution of linux platform.